### ISSUE TITLE
Each serializer

### ISSUE THREAD

**Issue by jareknowotka** [rails-api/active_model_serializers#664]
each_serializer options seems not to work in 0.9.0

```
render json: collection, serializer: false, each_serializer: EachSerializer
```
insead of specified serializer, it just returns serialised object.

after downgrading to 0.8.2 everything works as expected.

**Comment by fsainz** [comment]
I'm sure there's a better way but after struggling with it I ended up following this pattern:

in this case to handle the serialization of attachments

```ruby
serializer = ActiveModel::Serializer::ArraySerializer.new(attachments, each_serializer:AttachmentSerializer)
render json: ActiveModel::Serializer::Adapter::JsonApi.new(serializer).as_json
```

**Comment by dfjosh** [comment]
each_serializer still not fixed in 0.9.3, I downgraded to 0.8.3 and it works

**Comment by bf4** [comment]
@dfjosh If you have some test code I could run, I could probably help.  See https://github.com/rails-api/active_model_serializers/blob/master/CONTRIBUTING.md
